CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ endif()
 #
 # Dependencies
 #
-# FetchContent's composibility isn't very good. We must include the
+# FetchContent's composability isn't very good. We must include the
```

Comment:
nit: copyright

Reply:
Thanks for the review! The copyright looks up-to-date. Did I miss something?

Reply:
sorry. My bad I assumed all the years need to be changed.